src/Card/README.md

```diff
@@ -489,7 +563,8 @@ behavior.
   }}
 >
   <Card>
-    <Card.ImageCap 
+    <Card.ImageCap
+      skeletonHeight={140}
```

Comment:
If this `Card` isn't in a loading state (no `isLoading` prop), do we need the `skeletonHeight` prop here?